snippets/logging-restructure-snip.adoc

```diff
@@ -0,0 +1,13 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+
+[NOTE]
+====
+For Logging 5.5 and higher, documentation is organized by version.  
```

Comment:
Not sure what this means? I think we have docs for all released Logging versions previously?

Reply:
This is a snippet for future use in prior versions of documentation in support of restructuring the documentation.